lib/src/widgets/stars.dart

```diff
@@ -35,6 +35,7 @@ class StarRating extends StatelessWidget {
       color: color,
       borderColor: color,
       size: size,
+      isReadOnly: true,
```

Comment:
In newer versions default value of `isReadOnly` is `false` so it was possible to change stars when it shouldn't, the rating dialog itself is called in `review_create.dart line 49` where `SmoothStarRating `widget is called directly

Reply:
I noticed this one too, thanks for fixing!

Reply:
Ah, I think they used to determine if this was readOnly or not, depending on it having (or not) a change handler.

Reply:
(it is also presented in `begin` branch)

Reply:
I can backport this fix to `begin`, don't worry about that!

Reply:
(Done)